python/nano/test/pytorch/tests/test_inference_optimizer.py

```diff
@@ -382,6 +382,25 @@ def test_multi_instance(self):
             np.testing.assert_allclose(pred1, pred2, atol=1e-4,
                                         err_msg=f"\npred1: {pred1}\npred2: {pred2}\n")
 
+    def test_multi_instance_with_specify_cores(self):
+        model = Net()
+        model.eval()
+        inference_opt = InferenceOptimizer()
+        multi_instance_model = inference_opt.to_multi_instance(model, num_processes=2,
+                                                               cpu_for_each_process=[[0], [1]])
```

Comment:
I think you should allow the user to simply specify `cores_per_process`, instead of a map of cores for each process

Reply:
done